src/app/courses-dashboard/manage-course/manage-course.component.html

```diff
@@ -8,37 +8,80 @@ <h2>{{ editCourseId ? 'Edit Course' : 'New Course' }}</h2>
       class="form-column manage-course-form"
       (ngSubmit)="onSubmit()"
     >
-      <div class="form-label">
-        <label for="title">Course Name:</label>
-        <span class="form-hint">*required</span>
+      <div class="form-field">
+        <div class="form-label">
+          <label for="title">Title:</label>
+          <span class="form-error" *ngIf="isError('name')">
+            {{ outputError('name') }}
```

Comment:
using method invocation in templates is considered a bad practice and can cause performance issues.
In such cases better use pipes.